lib/netext/httpext/request_test.go

```diff
@@ -207,7 +211,7 @@ func TestResponseStatus(t *testing.T) {
 					ResponseType: ResponseTypeNone,
 				}
 
-				res, err := MakeRequest(ctx, preq)
+				res, err := MakeRequest(context.Background(), state, preq)
```

Comment:
can this be any context that will be canceled/timeouted in some time

Reply:
Resolved by the [latest force push](https://github.com/grafana/k6/compare/f072e4d180fb41c3686dd0c44a6ae8fbd0783193..18919f3f0f95645ace7c0a7ba1f71423978509e8)